content/en/blog/2022/introducing-ambient-mesh/index.md

```diff
@@ -137,3 +137,5 @@ We would love your feedback to help shape the solution.
 A build of Istio which supports ambient mesh is available to [download and try](/blog/2022/get-started-ambient/) in the [Istio Experimental repo]({{< github_raw >}}/tree/experimental-ambient).
 A list of missing features and work items is available in the [README]({{< github_raw >}}/blob/experimental-ambient/README.md).
 Please try it out and let us know what you think!
+
+*Contributors: Aaron Birkland (Solo.io), Craig Box (Google), Greg Hanson (Solo.io), Daniel Hawton (Solo.io), John Howard (Google), Ethan J. Jackson (Google), Denis Jannot (Solo.io), Abhi Joglekar (Google), Yuval Kohavi (Solo.io), Steven Landow (Google), Idit Levine (Solo.io), Oliver Liu (Google), Yossi Mesika (Solo.io), Justin Pettit (Google), Neeraj Poddar (Solo.io), Nina Polshakova (Solo.io), Christian Posta (Solo.io), Doug Reid (Google), Louis Ryan (Google), Lin Sun (Solo.io), Eitan Yarmush (Solo.io), Kuat Yessenov (Google), Francis Zhou (Google)*
```

Comment:
```suggestion
*Thank you to the team that contributed to the launch of ambient mesh: Aaron Birkland, Craig Box, Greg Hanson, Daniel Hawton, John Howard, Ethan J. Jackson, Denis Jannot, Abhi Joglekar, Yuval Kohavi, Steven Landow, Idit Levine, Oliver Liu, Yossi Mesika, Justin Pettit, Neeraj Poddar, Nina Polshakova, Christian Posta, Doug Reid, Louis Ryan, Lin Sun, Eitan Yarmush, Kuat Yessenov and Francis Zhou*
```

Reply:
That list is an eyesore with the affiliations.  With so many names, it's also better for the community to adjust it in the proposed fashion.

Reply:
I can take your other suggestions but i think we have agreed to put company names.